eng/packages/http-client-csharp-mgmt/emitter/src/resource-detection.ts

```diff
@@ -215,6 +221,23 @@ function isCRUDKind(kind: ResourceOperationKind): boolean {
   ].includes(kind);
 }
 
+function validateResourceGetMethods(
+  sdkContext: CSharpEmitterContext,
+  resourceMetadata: ResourceMetadata
+): void {
+  const getMethods = resourceMetadata.methods.filter(
+    (m) => m.kind === ResourceOperationKind.Get
+  );
+  
+  if (getMethods.length > 1) {
+    $lib.reportDiagnostic(sdkContext.program, {
+      code: "duplicate-get-method",
+      format: { resourceName: resourceMetadata.resourceName },
+      target: NoTarget
+    });
+  }
```

Comment:
I think in the message, we should include the op or fully qualified identifiers of those duplicated operations

Please also try to find the target, we could use the op of the first duplicated method.